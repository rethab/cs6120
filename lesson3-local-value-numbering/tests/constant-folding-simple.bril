@main {
  a: int = const 4;
  b: int = const 2;
  sum: int = add a b;
  print sum;
}
