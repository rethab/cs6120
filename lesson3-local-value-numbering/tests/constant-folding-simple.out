@main {
  sum: int = const 6;
  print sum;
}
